paddle/fluid/pybind/tensor.cc

```diff
@@ -393,7 +393,7 @@ void BindTensor(pybind11::module &m) {  // NOLINT
         Examples:
             .. code-block:: python
 
-                import paddle.fluid as fluid
+                import paddle.base as fluid
```

Comment:
is it better to use `import paddle.base as base`, and replace `fluid.Tensor()` with `base.Tensor()` ?

Reply:
Can other example code in this file be processed in the same way

Reply:
Yes, works of Document Standardization will be done in open-source task: https://github.com/PaddlePaddle/Paddle/issues/55629